pytorch_lightning/trainer/configuration_validator.py

```diff
@@ -290,3 +292,18 @@ def _check_dl_idx_in_on_train_batch_hooks(trainer: "pl.Trainer", model: "pl.Ligh
                     f"Base `Callback.{hook}` hook signature has changed in v1.5."
                     " The `dataloader_idx` argument will be removed in v1.7."
                 )
+
+
+# TODO: Delete _check_on_hpc_hooks in v1.8
+def _check_on_hpc_hooks(model: "pl.LightningModule") -> None:
+    if is_overridden("on_hpc_save", model):
+        rank_zero_deprecation(
+            "Method `model.on_hpc_save` is deprecated in v1.6 and"
+            " will be removed in v1.8. Please use `model.on_save_checkpoint` instead."
+        )
+
+    if is_overridden("on_hpc_load", model):
+        rank_zero_deprecation(
+            "Method `model.on_hpc_load` is deprecated in v1.6 and"
+            " will be removed in v1.8. Please use `model.on_load_checkpoint` instead."
```

Comment:
I think we generally refer to the class name within deprecation warnings instead of a variable name.
`model.on_load_checkpoint` -> `LightningModule.on_load_checkpoint` for both